design-docs/content-gsx-notes.md

```diff
@@ -0,0 +1,52 @@
+# Create a Getting Started Experience (GSX)
+
+We'd like to make sure folks who come to the docs site can get started easily. We'd like them to know how to:
+
+* Get an API key
+* Authenticate against the service
+* Search and download data with the cURL CLI
+* Search and download data with the SDK
+* Find key content and support
+
+# A clear top nav will help with the GSX
+
+## Add a clear top nav to mkdocs materials using a custom home page
+
+[" if the theme.custom_dir configuration value is used in combination with an existing theme, the theme.custom_dir can be used to replace only specific parts of a built-in theme"](https://www.mkdocs.org/dev-guide/themes/)
+
+["built-in themes are implemented in a base.html file, which main.html extends"](https://www.mkdocs.org/dev-guide/themes/)
+
+### Create a custom theme
```

Comment:
this seems nice, absolutely. My question is, where does a custom theme come in for priorities and what is the LOE?

Reply:
concerns I have are 1) too custom means users have to orient to a new paradigm and 2) it's an additional chunk of code to maintain